Userland/Libraries/LibC/netdb.cpp

```diff
@@ -825,4 +829,32 @@ int getnameinfo(const struct sockaddr* __restrict addr, socklen_t addrlen, char*
 
     return 0;
 }
+
+void herror(char const* s)
+{
+    int saved_herrno = h_errno;
+    dbgln("herror(): {}: {}", s, hstrerror(saved_herrno));
+    warnln("{}: {}", s, hstrerror(saved_herrno));
+}
+
+char const* hstrerror(int err)
+{
+    switch (err) {
+    case HOST_NOT_FOUND:
+        return "The specified host is unknown.";
+        break;
+    case NO_DATA:
+        return "The requested name is valid but does not have an IP address.";
+        break;
+    case NO_RECOVERY:
+        return "A nonrecoverable name server error occurred";
+        break;
+    case TRY_AGAIN:
+        return "A temporary error occurred on an authoritative name server. Try again later.";
+        break;
+    default:
+        return "Unknown error.";
+        break;
```

Comment:
The `break`s here aren't required (since it `return`s before it can ever `break`)

Reply:
I will remove them.